src/indexing.jl

```diff
@@ -379,3 +379,8 @@ Base.unsafe_view(A::AbstractArray, i1::StaticIndexing, indices::StaticIndexing..
 # instead of unsafe_view so we need this method too (Base._maybe_reindex
 # is a good example)
 Base.SubArray(A::AbstractArray, indices::NTuple{<:Any,StaticIndexing}) = Base.SubArray(A, map(unwrap, indices))
```

Comment:
Isn't it sufficient to restrict the second argument to non-empty tuples here? I.e. `Tuple{StaticIndexing,Vararg{StaticIndexing}}`?

Reply:
I didn't know I can put `Vararg` there, good idea.

Reply:
I've updated the PR, it should be good now.